integration/setup/provider.go

```diff
@@ -2,42 +2,9 @@ package setup
 
 import (
 	"context"
-
-	"github.com/giantswarm/e2e-harness/pkg/release"
-	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
-	"github.com/giantswarm/microerror"
-
-	"github.com/giantswarm/azure-operator/integration/env"
 )
 
 // provider installs the operator and tenant cluster CR.
 func provider(ctx context.Context, config Config) error {
-	{
-		c := chartvalues.CredentialdConfig{
```

Comment:
I believe `credentiald` was only used to get the `credential-default` secret created. I removed this code and created the secret manually.